### PR TITLE
Adding checks to avoid 'Uncaught SyntaxError: Unexpected end of input'

### PR DESCRIPTION
**Pull request by gangeshwari-rajavelu**

### CODE REVIEW

File: src/readJsonLines.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 function readJsonLines(string) {
   return string.split('\n')
     .filter(line => line)
+    .filter(line => line.startsWith('{')
+    .filter(line => line.endsWith('}')
```
Comment (oprogramador) [inline src/readJsonLines.js:5]: could you cover this with some tests?

File: src/readJsonLines.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 function readJsonLines(string) {
   return string.split('\n')
     .filter(line => line)
+    .filter(line => line.startsWith('{')
```
Comment (oprogramador) [inline src/readJsonLines.js:4]: a more reliable way for checking valid JSON is to try to parse it and handle an error in a `catch` block

File: src/readJsonLines.js (hunk 1 above)
Comment (gangeshwari-rajavelu) [reply src/readJsonLines.js:5]: sure

File: src/readJsonLines.js (hunk 2 above)
Comment (gangeshwari-rajavelu) [reply src/readJsonLines.js:4]: okay, will check this

**Comment by oprogramador** [comment]
@gangeshwari-rajavelu 

I implemented that in this commit 8a84e188206709e039b3a9ed1bfc40e16f99b740, handling any invalid JSON and providing tests.

**Pull request closed without merging**